File: ecos-records/src/main/java/ru/citeck/ecos/records3/record/operation/delete/DelStatus.java
package ru.citeck.ecos.records3.record.operation.delete;

public enum DelStatus {
    /**
     * Deletion was successful
     */
    OK,
    /**
     * Internal error
     */
    ERROR,
    /**
     * Record is protected from deletion
     */
    PROTECTED,
    /**
     * Operation is not supported
     */
    NOT_SUPPORTED
}

File: ecos-records/src/main/java/ru/citeck/ecos/records3/record/dao/delete/DelStatus.kt
package ru.citeck.ecos.records3.record.dao.delete

enum class DelStatus {
    /**
     * Deletion was successful
     */
    OK,

    /**
     * Internal error
     */
    ERROR,

    /**
     * Record is protected from deletion
     */
    PROTECTED,

    /**
     * Operation is not supported
     */
    NOT_SUPPORTED
}
